src/sidebar/build-thread.js

```diff
@@ -1,4 +1,22 @@
-/** Default state for new threads, before applying filters etc. */
+/**
+ * @typedef {import('../types/api').Annotation} Annotation
```

Comment:
I believe you can use `import(...)` in any place where a type is expected, so it would work with `@param {...}` or `@prop {...}` as well. The use of `@typedef` here is just to define a local alias to save repeating the whole `import(...).Annotation` several times.